### ISSUE TITLE
Incorrect character in fr.po file

### ISSUE THREAD

**Issue by undef21** [ihhub/fheroes2#5503]
### Preliminary checks

- [X] I've checked that there aren't [**other open issues**](https://github.com/ihhub/fheroes2/issues?q=is%3Aissue) on the same topic.
- [X] I've checked that this issue is reproducible on the [**latest snapshot build**](https://ihhub.github.io/fheroes2/INSTALL.html#snapshots-latest-builds).

### Platform

Linux

### Describe the bug

fr.po is the modified ASCII, but it contains [non-ASCII and non-mapped char Ç](https://github.com/ihhub/fheroes2/blob/cf315a9c7df9a2a996ece91e94b46b6979ca3899/files/lang/fr.po#L4853).
Thanks @hgdagon for finding it.

### Save file

None

**Comment by zenseii** [comment]
Hi @undef21! I couldn't quickly see an example of that letter by opening the link, but I take it that you are referring specifically to the uppercase C with cedilla Ç since the lowercase is generated and is shown in-game, afaik?

I'm not sure if you have a proposal for solution, but since the current characters present for French, both generated and when provided with French assets, reflect what you get in the original French Homm2 experience I'm inclined to call this issue an improvement?

As for a solution, I figure moving French to CP1252 generation and replacing use of original extra assets for French fonts with our generated ones, could be one approach. This would be logical in the case of missing speicial fonts in the original assets like uppercase C with cedilla `Ç` and if we end up adding `Œ` and `œ`.

**Comment by undef21** [comment]
Yes, the problem char is `Ç`.
The problem block:
```
msgid ""
"It doesn't help to pray twice before a battle. Come back after you've fought."
msgstr ""
"Ça ne sert à rien de prier deux fois avant une bataille. Revenez après avoir "
"combattu."
```

Original French assets and generated font dont't have this char so in game this one is replaced with space. I don't know how properly transliterate this char to ASCII. Possible solutions:
- replace `Ç` with `C`
- replace `Ç` with presented `ç`.

> As for a solution, I figure moving French to CP1252 generation and replacing use of original extra assets for French fonts with our generated ones, could be one approach.

Whatever French maps contain text in this custom encoding. We'll be need to recode this strings to CP1252.

**Comment by undef21** [comment]
@zenseii i update description. It's definitely bug.

**Comment by zenseii** [comment]
> @zenseii i update description. It's definitely bug.

Yeah, since it gets replaced with a white space it's a bug, and we should at least replace it with uppercase C or lowercase ç for the moment until someone writes code to generate uppercase Ç. It would be interesting to see what the OG does here.
